code/WorkInProgress/AzrunStuff.dm

```diff
@@ -43,20 +43,20 @@
 
 /datum/manufacture/sub/treads
 	name = "Vehicle Treads"
-	item_paths = list("MET-2","CON-1")
-	item_amounts = list(5,2)
+	item_requirements = list("metal_dense" = 5,
+							 "conductive" = 2)
```

Comment:
![image](https://github.com/goonstation/goonstation/assets/62817778/4e910303-5b32-45ee-aef4-a5303bcb32b2)
I am absolutely not going to require this change, but this is a classic "indent with tabs, align with spaces" thing.

Reply:
i think you're wrong here tbh mordent, this is javascript poisoning your brain - we use tabs for this all over the codebase and in modern code, it's just our fault we don't have a standard tab width